AlexNet.html

```diff
@@ -129,6 +129,10 @@ <h4>Style:</h4>
                                 <input type="color" value="#ffbbbb" id="color3">
                                 <label for="color3">Color 3</label>
                             </div>
+                            <div>
+                                <label for="fontScale">Font Size</label>
+                                <input type="range" id="fontScale" name="" min="1" max="3" step="0.2" value="1" style="position: relative; top: 3px;">
+                            </div>
```

Comment:
Could you move this to below "Show Conv Dimensions"

Reply:
Of course